www/js/util.js

```diff
@@ -3112,3 +3060,95 @@ function stopQueueSpinner(data) {
         $("#queueprogress").remove();
     }
 }
+
+function CSEmoteList(selector) {
+    EmoteList.call(this, selector);
+}
+
+CSEmoteList.prototype = Object.create(EmoteList.prototype);
+
+CSEmoteList.prototype.loadPage = function (page) {
+    var tbody = this.table.children[1];
+    tbody.innerHTML = "";
+
+    var start = page * this.itemsPerPage;
+    if (start >= this.emotes.length) {
+        return;
+    }
+    var end = Math.min(start + this.itemsPerPage, this.emotes.length);
+    var self = this;
```

Comment:
Closing the edit dialogue jumps you back to page 1. :rage: That's gonna piss me off (and probably others) to no end. It's especially annoying since it happens even if you haven't modified it.

`    var self = this; this.page = page;`